src/sentry/integrations/bitbucket/integration.py

```diff
@@ -114,22 +123,17 @@ def get_pipeline_views(self):
         return [identity_pipeline_view, VerifyInstallation()]
 
     def post_install(self, integration, organization):
-        repos = Repository.objects.filter(
+        repo_ids = Repository.objects.filter(
             organization_id=organization.id,
             provider='bitbucket',
-        )
-
-        unmigrateable_repos = self \
-            .get_installation(integration, organization.id) \
-            .get_unmigratable_repositories()
-
-        for repo in filter(lambda r: r not in unmigrateable_repos, repos):
-            repo.update(
-                integration_id=integration.id,
-                provider='integrations:bitbucket',
-            )
-
-        PluginMigrator(integration, organization).call()
+        ).values_list('id', flat=True)
+
+        for repo_id in repo_ids:
+            migrate_repo.apply_async(kwargs={
+                'repo_id': repo_id,
+                'integration_id': integration.id,
+                'organization_id': organization.id,
+            })
```

Comment:
since we're limiting to `provider='bitbucket'` for repos, i think the only ones we'd catch with an integration id are ones where we'd want to update the provider anyway (to `integrations:bitbucket`). i could do something like add logging if the integration ids didn't match but i think that would be unlikely unless there was something funky going on

Reply:
Got it. Yea, logging could be useful.